card.css

```diff
@@ -0,0 +1,21 @@
+@import "base.css";
+
+.videoBox {
+  width: 370px;
+  height: 370px;
```

Comment:
This might be depending on the size of the video, but still variable size is more responsive than fixed :)

Reply:
!!! Change planed in dedicated user story for UI responsiveness

Thank you